apps/site/pages/docs/getting-started/1-faststore-onboarding/dashboard.mdx

```diff
@@ -11,6 +11,69 @@ import { Callout } from 'nextra-theme-docs'
+| **Deployment**        | Opens the deployed version of the store in a new tab.                                                                                 |
+| **Domain**            | Opens the URL of your store's domain, which is the same as the preview.                                                               |
+| **Last commit**       | Displays the most recent commit made to your store's repository, including the date, month, year, and the person who made the commit. |
+| **Status**            | Displays the status of the deployment.                                                                                                |
+
+The **Status** field can have one of the following values: 
+- `Success`: The deployment was successful. If you want to check the commit associated with it, go to the **Deploys** tab.
+
+- `Error`: There was an error in the pipeline that caused the deployment to fail, but the build can still be successful. 
+To understand what happened, go to the **Deploys** tab, check the last commit, and open the repository for further investigation.
+
+--- 
+
+## Deploys
+
+The **Deploys** view provides a summary of each deployment made in your store's repository.
+
+![deploy-view](https://vtexhelp.vtexassets.com/assets/docs/src/dashboard-view-deploys___e5f04f3e5613d52e4b2864ef48134608.png)
+
+| Field                         | Description                                                                                                                  |
+| :---------------------------- | :--------------------------------------------------------------------------------------------------------------------------- |
+| **Preview URL**               | Displays a store preview for a specific commit.                                                                                          |
+| **Commit**                    | Displays the commit hash associated with a particular commit.                                                                                     |
+| **Timestamp**                 | Displays the date and time of the commit in the format `dd/mm/yyyy`.                                                     |
+| **Elapsed time**              | Displays the time it took to complete the commit process.   |
+| **Deploy**                    | Indicates the status of the deployment. Possible statuses are `Success` and `Error`. For more information on the `Error` status, go to the repository and check what failed in the commit.                   |
+| **Quality checks**            | Indicates the status of the quality checks (Lighthouse audits). Possible statuses are `Success` and `Error`. For more information on the `Error` status, go to the repository and check what failed in the commit. *Remember: Lighthouse audits are an essential step in the [go-live checklist](/docs/go-live/overview#go-live-checklist).* Ensure you achieve the recommended score for the lighthouse checks to pass.                     |
```

Comment:
We've just added a new status `Skipped`. Can you add it here pls?

More details on this PR: https://github.com/vtex-sites/faststore.admin/pull/41